creation/create_cvmfsexec_distros.sh

```diff
@@ -3,123 +3,166 @@
 # SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 # SPDX-License-Identifier: Apache-2.0
 
+# Hardcoded variables
+CVMFSEXEC_REPO="https://www.github.com/cvmfs/cvmfsexec.git"
+DEFAULT_WORK_DIR="/var/lib/gwms-factory/work-dir"
+
 usage() {
-	echo "This script is used to generate cvmfsexec distributions for all"
-	echo "supported machine types (platform- and architecture-based)."
-	echo "The script takes one parameter {osg|egi|default} which specifies"
-	echo "the source to download the latest cvmfs configuration and repositories."
+	echo "This script generates cvmfsexec distributions for various cvmfs configurations and supported machine types."
+	echo "Usage: $0 [--work-dir DIR] SOURCES_LIST PLATFORMS_LIST"
+	echo ""
+	echo "DIR: full, absolute path to the factory work directory"
+	echo ""
+	echo "SOURCES_LIST: specifies the source(s) to download the latest cvmfs "
+	echo "configuration and repositories from. Must be at least one value or "
+	echo "a comma-separated list of values from the options {osg|egi|default}."
+	echo ""
+	echo "PLATFORMS_LIST: indicates the machine types (platform- and architecture-based) "
+	echo "for which distributions is to be built. Must be at least one value or "
+	echo "a comma-separated list of values from the options {rhel7-x86_64|rhel8-x86_64|suse15-x86_64}."
```

Comment:
Conceptually, it can be empty; in which case, PLATFORMS_LIST will be all available platforms (default). I wrote the usage documentation, from the perspective where it will be executed as a standalone script. I remember you had mentioned that I should also consider the script being run by itself and not always being invoked from a factory reconfig standpoint. I shall update the usage documentation to reflect what I described above and change the main part of the script accordingly.